pretext/ExtraTopics/BinaryRepresentations.ptx

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-binaryrepresentations">
+  <title>Binary Representations</title>
+
+  <p>Have you ever seen all of the "hacker" 01010110101010s in the movies? As you probably
+    know, this is called binary. But it's not really how modern day hacking works. However,
+    it is the basis of all of computing, and although you won't be typing 0s and 1s at a
+    keyboard all day, it's useful to know.</p>
+
+  <p>Quick background: binary is a numbering system, just like decimal (the numbering system we use). 
+    Decimal uses the digits 0-9, but binary only uses the digits 0 and 1, which are called bits.</p>
```

Comment:
I would maybe like to see a connection like "expanded form" in decimal and maybe how that is similar to binary, except that its base 2 rather than base 10.